routing/osrm_data_facade.hpp

```diff
@@ -272,6 +272,10 @@ template <class EdgeDataT> class OsrmDataFacade : public OsrmRawDataFacade<EdgeD
   {
     Clear();
 
+    // Load huge data first.
```

Comment:
How about to clarify why we map largest file first:
```c++
// Map huge data first, as we hope it will reduce fragmentation of the program address space.
```